config/development.json

```diff
@@ -19,6 +19,8 @@
 		"business_group_id": false
 	},
 	"signup_url": "/start",
+	"desktop_oauth_client_id": "",
+	"desktop_oauth_client_secret": "",
```

Comment:
These have been moved to `secrets.json` so we don't need them directly in config files anymore.